agenta-web/src/components/pages/observability/drawer/TestsetDrawer/TestsetDrawer.tsx

```diff
@@ -245,10 +275,10 @@ const TestsetDrawer = ({onClose, data, ...props}: TestsetDrawerProps) => {
     }
 
     useUpdateEffect(() => {
```

Comment:
we should stop using this hook. according to docs:

> @deprecated
>
> Don't use this hook, it's an anti-pattern. Custom hook that runs an effect only on updates (not on the initial render).

Reply:
fixed